cmd/osm-bootstrap/crds/config_mesh_root_certificate.yaml

```diff
@@ -85,13 +85,17 @@ spec:
                       type: object
                       required:
                         - host
+                        - port
                         - role
                         - protocol
                         - token
                       properties:
                         host:
                           description: Host name for the Vault server
                           type: string
+                        port:
```

Comment:
Should vault.port also be added to `values.yaml` and `values.schema.json`?

Reply:
done!